src/components/edit-asset-address-form/editable-address/editable-address.tsx

```diff
@@ -46,7 +46,43 @@ export const EditableAddress = ({
   setErrorDescription,
   setCurrentAssetAddress,
 }: EditableAddressProperties): JSX.Element => {
-  const [submitEditEnabled, setSubmitEditEnabled] = useState<boolean>(true);
+  const [addressEditSuccessful, setAddressEditSuccessful] = useState<boolean>(false);
+
+  const renderFormActionButtons = () => {
+    if (!addressEditSuccessful) {
+      return (
+        <>
+          <div className="form-actions">
+            <button
+              className="govuk-button lbh-button"
+              data-module="govuk-button"
+              type="submit"
+              id="submit-address-button"
+            >
+              Update to this address
+            </button>
+
+            <RouterLink
+              to={`/property/${assetDetails.id}`}
+              className="govuk-button govuk-secondary lbh-button lbh-button--secondary"
+            >
+              Cancel
+            </RouterLink>
+          </div>
+        </>
+      );
+    }
+    return (
+      <>
+        <RouterLink
+          to={`/property/${assetDetails.id}`}
+          className="govuk-button lbh-button"
+        >
+          Back to asset view
+        </RouterLink>
+      </>
+    );
+  };
```

Comment:
Should the bottom butttons include the form-actions class

Reply:
Good point, will change